temporenc.rst

```diff
@@ -333,7 +329,8 @@ byte of the encoded value. The advantages of this approach are:
 * The total size of encoded values is very small.
 
 * Encoded values of the same *type* (and precision) can be sorted
-  lexicographically.
+  lexicographically resulting in chronological ordering, except when
+  mixing values with different time zone offsets.
```

Comment:
I think this might be unclear to someone who didn't have more context on the rest of the spec.

What about something like:
> Lexicographical sorting of encoded values of the same *type* (and precision) results in the same ordering as the natural text-based ordering. This means that mixing values with different UTC offsets will sort based on the literal date and time specified, not the equivalent instant in UTC.